pkg/doc/jose/decrypter.go

```diff
@@ -37,6 +36,8 @@ type JWEDecrypt struct {
 	store  storage.Store
 	crypto cryptoapi.Crypto
 	kms    kms.KeyManager
+	// ForGoJoseInterop similar to JWEEncrypt, adds extra encoding for APU/APV for interop with Go Jose.
```

Comment:
I'm not sure about this.

Reply:
removed flag, JWE encryption/decryption is compliant with go-jose